report_xlsx/__manifest__.py

```diff
@@ -10,6 +10,7 @@
     "development_status": "Production/Stable",
     "license": "AGPL-3",
     "depends": ["base", "web"],
+    "external_dependencies": {"python": ["xlsxwriter"]},
```

Comment:
Why changing this?

Reply:
I could see that `report_xlsx` was using the `xlsxwriter` library already (https://github.com/OCA/reporting-engine/blob/13.0/report_xlsx/report/report_abstract_xlsx.py#L13), but the external dependency was not explicitly in the manifest file. I thought that this should be added so that the library is also added to the `requirements.txt` file. Perhaps this is not needed?

Reply:
It's already in Odoo requirements, so that's why is not needed.

Reply:
Oh, I see. I will remove it then. Thank you 😊